src/routes/data/transformers.rs

```diff
@@ -98,3 +112,24 @@ where
 
     Ok(hash_data)
 }
+
+pub fn validate_card_metadata(
+    stored_payload: &Option<storage::types::Locker>,
```

Comment:
nit: we generally should follow this type of references, i.e. Option of a reference rather than a reference to a option as option itself doesn't have size.
```suggestion
    stored_payload: Option<&storage::types::Locker>,
```